src/core/evaluator.js

```diff
@@ -1009,16 +1009,23 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           // var x = pt[0];
           // var y = pt[1];
 
+          var charSpacing = 0;
+          if (i > 0 || textChunk.str.length > 0) {
```

Comment:
That clears it up, thanks. @CodingFabian Are you also okay with removing `i > 0`?